Assets/Mirror/Editor/Weaver/Processors/RpcProcessor.cs

```diff
@@ -81,6 +81,8 @@ public static MethodDefinition ProcessRpcCall(TypeDefinition td, MethodDefinitio
             rpcWorker.Append(rpcWorker.Create(OpCodes.Ldc_I4, NetworkBehaviourProcessor.GetChannelId(ca)));
             rpcWorker.Append(rpcWorker.Create(OpCodes.Callvirt, Weaver.sendRpcInternal));
 
+            NetworkBehaviourProcessor.WriteRecycleWriter(rpcWorker);
```

Comment:
@vis2k  check line 62 in this file,  i calls WriteCreateWriter.   That was already there,  but it now calls the pool instead of creating a new object